src/main/java/pl/javastart/task/Main.java

```diff
@@ -11,6 +14,53 @@ public static void main(String[] args) {
 
     public void run(Scanner scanner) {
         // uzupełnij rozwiązanie. Korzystaj z przekazanego w parametrze scannera
+
+        ZoneId localZone = ZoneId.of(java.util.TimeZone.getDefault().getID());
```

Comment:
To jest używane tylko wewnątrz metody getDateTime, więc możesz to zdefiniować tam.
Można też zapisać krócej `ZoneId.systemDefault();`